packages/react/src/components/MultiSelect/index.tsx

```diff
@@ -5,10 +5,5 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import MultiSelect from './MultiSelect';
-import { default as FilterableMultiSelect } from './FilterableMultiSelect';
-
-FilterableMultiSelect.displayName = 'FilterableMultiSelect';
-
-export { FilterableMultiSelect, MultiSelect };
-export default MultiSelect;
```

Comment:
Same thing here w/ removing the default export someone might (unlikely) be relying on.